source/sass/blocks/form-checkbox.scss

```
@@ -1,13 +1,20 @@
 .form-checkbox__title {
   font-size: 24px;
   line-height: 26px;
-  margin-bottom: 34px;
+  margin-bottom: 37px;
   padding: 0;
+  float: left;
+  z-index: 0;
 
   @media (min-width: $medium) {
     font-size: 36px;
     line-height: 36px;
     margin-bottom: 59px;
+    grid-column: span 2;
+  }
+
+  @media (min-width: $large) {
+    grid-column: span 4;
```

Тебе приходится переписывать количество колонок, потому что оно меняется при изменении ширины экрана, а тебе нужно, чтобы заголовок занимал все колонки.
Вопрос: а можно ли сделать иначе? Например, прям сказать ему "займи все колонки от первой до последней". Тогда не придется переписывать, при любом количестве заголовок растянется. И...

Конечно, можно!
Для этого нужно указать не через `span` заполнение, а начало и конец по линиям (от первой до последней):
```suggestion
    grid-column: 1 / -1;
```